charm.go

```diff
@@ -63,31 +64,37 @@ const (
 	SelectionContainers FormatSelectionReason = "containers"
 )
 
+var (
```

Comment:
Really need the braces here?

Reply:
I was going to add a bigger comment but didn't in the end.